### ISSUE TITLE
Feature Request: Fully automate viewcache clearing + database migration handling

### ISSUE THREAD

**Issue by berrnd** [grocy/grocy#2384]
After each and every release there tend to be more and more invalid Bug reports caused by a not emptied `data/viewcache` directory and/or not executed database migrations, so by people who can't read README for whatever reason: [How to update](https://github.com/grocy/grocy/tree/v4.0.3#how-to-update)

I still blame everyone opening a request which could simply be solved by reading README or doing a quick and easy search beforehand, but let's finally automate that, at least to reduce the support amount for myself.

What's stored in `data/viewcache` is (currently) dependent on / specific to the version and the config options `BASE_PATH` and `BASE_URL`. So let's somehow auto-clear that directory when any of this changes (normally that means an update happened).

And since new database migrations can only come with a new release, let's then take this also to fire off them.

**Comment by sur5r** [comment]
I was wondering whether this could be integrated into `update.sh` like this:

```bash
pushd public
php index.php
popd
```

From a quick test, this seems to work, but maybe I'm missing something.

**Comment by berrnd** [comment]
> but maybe I'm missing something.

Yes, each and everyone not using `update.sh`. Since Docker and related managed setups are the most popular ways to run Grocy, this won't be the solution and it's not the topic of this Feature Request.
